client/extensions/woocommerce/app/products/index.js

```diff
@@ -121,9 +121,9 @@ class Products extends Component {
 
 function mapStateToProps( state ) {
 	const site = getSelectedSiteWithFallback( state );
-	const productsLoaded = site && areProductsLoaded( state, 1, site.ID );
+	const productsLoaded = site && areProductsLoaded( state, { page: 1, per_page: 10 }, site.ID );
```

Comment:
We should probably make a product's `per_page` constant somewhere. If we ever want to change how many we display, we'll have a number of places to do it now.

Reply:
I do set a default in the code here, and I keep that default through the next PR.
Since per_page is a very UI sort of thing, I'd prefer to keep defaults for it in the React components. In later PRs (that I haven't put up just yet), I set a default prop in the component for per_page and use it there. That's the appropriate place to keep such values, IMO.